### ISSUE TITLE
Improve the exercise

### ISSUE THREAD

**Issue by oneeyedman** [Adalab/m1-evaluacion-intermedia-martagrio#1]
Really cool job @martagrio  !!!

Here you have some tips to improve the exercise:

## HTML
- [ ] Fix the language of the page

## CSS - structure
- [ ] Move the font CSS before your main CSS

## CSS - selectors
- [ ] Class selectors are free, let's use them instead tag selectors 
- [ ] Improve hierarchy adding more common styles to `body` tag as font-size, font-family, font-weight, color, background color

## CSS - box model
- [ ] Block elements like our main one do not need a width: 100% or width: 100vw property to tell them to do what they are going to do anyway ;)

## CSS - other
- [ ] Avoid  `BR` tags, we can transform the text with CSS; this way we keep concepts separated, HTML for content, and CSS for styles
- [ ] Fix color of links and main title size.

Let me know if you have any doubts ;)

Go for it!!

**Comment by martagrio** [comment]
Thank you so much for the tips!

I took them all into account and worked again on the assignment to improve it as you recommended. I created a new branch, fixed, and worked on it since I prefer to wait for your approval to make the final changes in the master.

With all the changes applied I think this could be the final version but of course feel free to add any more tips if you consider so.

Thanks again!

Kind Regards

P.S: just in case, I created a different branch, prueba, to apply a little change. Instead of using divs for each word in the main's title (h1) I just specified the width for h1 in both mobile and tablet/desktop versions so each word appears alone in a row and centered.
